src/css/components/_progress.scss

```diff
@@ -15,6 +15,7 @@
 .video-js .vjs-progress-control {
   @include flex(auto);
   @include display-flex(center);
+  margin: 0 0.45em 0 0.45em;
```

Comment:
It would seem that the Brightcove skin might need to include that in the source if it utilizes a fullscreen `SeekBar`?

Should video.js accommodate fullscreen `SeekBar` in its implementation?

Not sure best way to move forward on this.